packages/terra-overlay/src/LoadingOverlay.jsx

```diff
@@ -46,7 +46,7 @@ const defaultProps = {
   isOpen: false,
   backgroundStyle: BackgroundStyles.LIGHT,
   isRelativeToContainer: false,
-  rootSelector: '[data-terra-base]',
+  rootSelector: '#root',
```

Comment:
It seems we're making an assumption that the there is a root selector present. That assumption works for [terra-dev-site](https://github.com/cerner/terra-dev-site/blob/master/src/index.html#L11) and most orion apps, but it's not guaranteed that apps will have the root selector. If we want to change this from the base selector we might want to open it up to a prop but default to root and document it on the readme.

Reply:
We have [a prop for this already named rootSelector](https://github.com/cerner/terra-core/blob/master/packages/terra-overlay/src/LoadingOverlay.jsx#L35-L37). This line sets the default value based on that assumption that there is an `id="root"` in the DOM. For apps that don't have that id attribute, they can change it by passing a different selector to the `rootSelector` prop. I can update the prop type documentation to be more clear about this.

Reply:
If we already have a prop for this i don't think you have to document more. I should have looked more deeply for the doc.

Reply:
Updated the doc, the doc referenced a data attribute that was on terra-base div that used to be rendered. We've removed that div to [fix terra-base fill issues](https://github.com/cerner/terra-framework/issues/124), but with that, we needed to update the rootSelector we default to. https://github.com/cerner/terra-core/pull/2288/commits/c7e816f812722711252e67b6cb35d93fcb13e4bd